### PR TITLE
complete task 7 vote on an article

### PR DESCRIPTION
**Pull request by AmadReihun**
complete task 7 vote on an article

### CODE REVIEW

File: src/components/VoteArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      event.preventDefault();
+      setLikes((currLikes) => {
+        return currLikes - 1;
+      });
+      patchArticleVotes(article_id, -1);
+      setIsDisliked(true);
+      setIsLiked(false);
+    }
+  }
+
```
Comment (nc-christian-dutton) [inline src/components/VoteArticle.jsx:42]: nice work with these ☝️ , you could take it one step further and have one handleClick that does both, you'd need to find a way to pass a value to the handleClick that could be 1 or -1. Worth a look at some point 👍

File: src/utils/api.js [hunk 2]
```diff
@@ -12,3 +12,7 @@ export const fetchComments = (article_id = "") => {
   return ncApi.get(`/articles/${article_id}/comments`);
 };
 
+export const patchArticleVotes = (article_id, number) => {
+  return ncApi.patch(`/articles/${article_id}`, {inc_votes: number})
+}
+
```
Comment (nc-christian-dutton) [inline src/utils/api.js:18]: Nice job with this, only need one axios call that does both!

**Comment by nc-christian-dutton** [comment]
Good job! Working optimistically and I like that you can only vote by 1 in either direction

**Comment by AmadReihun** [comment]
Thanks Christian for your feedbacks

**Pull request merged**